### ISSUE TITLE
Some javascript tests aren't getting run by default

### ISSUE THREAD

**Issue by noisecapella** [mitodl/micromasters#2815]
For example `/static/js/components/email/lib_test.js`. I'm wondering if our file pattern is incorrect

**Comment by aliceriot** [comment]
If I do:

```
echo static/**/*_test.js | tr ' ' '\n' | wc -l
```

or

```
echo static/**/*/*_test.js | tr ' ' '\n' | wc -l
```

I get 97 test files. doing

```
tree static/js | grep _test.js | wc -l
```
returns the same number. The second glob pattern (`static/**/*/*_test.js`) is the one we use.

**Comment by aliceriot** [comment]
I don't see the file you mention in my working tree (`/static/js/components/email/lib_test.js`)

**Comment by aliceriot** [comment]
I think this is related to https://github.com/mitodl/micromasters/pull/2767, I reverted it and it matches the whole test suite again. Should have caught that in the review...

I am not enough of a bash expert to figure out what is wrong, @gsidebo do you have any ideas?

**Comment by gsidebo** [comment]
OK wow... Very confusing and frustrating. I'll look into it

**Comment by gsidebo** [comment]
If you print the command before it's run in `./js_test.sh`, it runs all 925 tests, but the command as it's `eval`'ed matches 600-something. This is definitely some bash chicanery

**Comment by aliceriot** [comment]
bash nooooo

**Comment by aliceriot** [comment]
@noisecapella I just checked, and if I run the same command in bash:

```
echo static/**/*/*_test.js | tr ' ' '\n' | wc -l
```

I get just 69! ack!

**Comment by noisecapella** [comment]
Yeah I feel like this is one of the few times where regexes would solve the problem rather than create it

**Comment by noisecapella** [comment]
This was closed by #2817
